<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="CubeViewWidget" name="openglWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Raster</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gbRendering">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string>Rendering</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Top Plane</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnSetRendererTopPlane">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Slices</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnSetRendererSlices">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbPlotting">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string>Show</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="5" column="1">
           <widget class="QSlider" name="sliderY">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="cbShowSliceZ">
            <property name="text">
             <string>Z Slice</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="spinBoxY"/>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QPushButton" name="btnResetGLView">
            <property name="text">
             <string>Reset Camera</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cbShowCube">
            <property name="text">
             <string>Cube</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Z Scaling</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="spinBoxX"/>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbShowBox">
            <property name="text">
             <string>Box</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="sliderX">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbShowTopPlane">
            <property name="text">
             <string>Top plane</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSlider" name="sliderZScale">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QSpinBox" name="spinBoxZScale">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="cbShowSliceY">
            <property name="text">
             <string>Y Slice</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QSpinBox" name="spinBoxZ"/>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="cbShowSliceX">
            <property name="text">
             <string>X Slice</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="sliderZ">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoadData">
           <property name="text">
            <string>Load Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionValidate">
   <property name="text">
    <string>Validate</string>
   </property>
  </action>
  <action name="actionLoadData">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="actionSetRendererTopPlane">
   <property name="text">
    <string>Set Top Plane Renderer</string>
   </property>
  </action>
  <action name="actionSetRendererSlices">
   <property name="text">
    <string>Set Slice Renderer</string>
   </property>
  </action>
  <action name="actionResetGLView">
   <property name="text">
    <string>Reset View</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CubeViewWidget</class>
   <extends>QWidget</extends>
   <header>snippets.qgisrenderer.volumetric</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowSliceX</sender>
   <signal>clicked(bool)</signal>
   <receiver>sliderX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowSliceX</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBoxX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowSliceY</sender>
   <signal>clicked(bool)</signal>
   <receiver>sliderY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowSliceY</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBoxY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowSliceZ</sender>
   <signal>clicked(bool)</signal>
   <receiver>sliderZ</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowSliceZ</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBoxZ</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderZScale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxZScale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxZScale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderZScale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
